resources/translations.qrc

```diff
@@ -1,5 +1,6 @@
 <RCC>
     <qresource prefix="/">
         <file>i18n/kiwix-desktop_fr.qm</file>
+        <file>i18n/en.json</file>
```

Comment:
Do we still need `i18n/kiwix-desktop_fr.qm` ? All application translation should be in the json file no ?